### PR TITLE
Updates gids.R to use master_scoreboard.json instead of root day directory

### PR DESCRIPTION
**Pull request by james-ingold**
The root game day directory now returns a 404, for example, https://gd2.mlb.com/components/game/mlb/year_2020/month_02/day_26/

This change uses master_scoreboard.json to get game ids and updates gids through 2020.

### CODE REVIEW

File: R/scrape.R [hunk 1]
```diff
@@ -459,10 +459,12 @@ fill.NAs <- function(value, fields) {
...
   message("grabbing new game IDs")
-  scoreboards <- paste0(makeUrls(start=last.date, end=end, gids=""), "/epg.xml")
+  scoreboards <- paste0(makeUrls(start=last.date, end=end, gids=""), "/miniscoreboard.xml")
+  #scoreboards <- paste0(makeUrls(start=last.date, end=end, gids=""), "/epg.xml")
   obs <- XML2Obs(scoreboards) #Note to future self -- using `xpath='//game[@gameday_link]'` for future games gives the RCurl error -- Recv failure: Connection reset by peer
-  obs2 <- obs[grep("^epg//game$", names(obs))]
-  gids <- collapse_obs(obs2)[,"gameday"]
+  obs2 <- obs[grep("^games//game$", names(obs))]
+  #obs2 <- obs[grep("^epg//game$", names(obs))]
+  gids <- collapse_obs(obs2)[,"gameday_link"]
```
Comment (cpsievert) [inline R/scrape.R]: Hmm, I don't see a `gameday_link` field...seems like `updateGids` should be doing something like `inst/scripts/gids.R`, though (i.e., shouldn't it be pulling from `master_scoreboard.json` now?)

Comment (james-ingold) [reply R/scrape.R]: Agreed, this was just a pull request to get gids.R working again. I'm still working on making scrape work in 2020. I can hold off making a pr until then, thanks for reviewing!

Comment (james-ingold) [reply R/scrape.R]: @cpsievert should be good to go now

File: inst/scripts/gids.R [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
+day_list <- lapply(scoreboards, fromJSON)
+
 new_gids <- lapply(day_list, function(day) {
-  day %>%
-    html_nodes("a") %>%
-    html_text() %>%
-    stringr::str_trim() %>%
-    stringr::str_remove("/$") %>%
-    grep("^gid_", ., value = TRUE)
+  payload <- day$data$games$game$gameday
```
Comment (cpsievert) [inline inst/scripts/gids.R]:
```suggestion
  day$data$games$game$gameday
```

File: R/scrape.R [hunk 3]
```diff
@@ -459,10 +459,12 @@ fill.NAs <- function(value, fields) {
...
 updateGids <- function(last.date, end) {
   message("grabbing new game IDs")
-  scoreboards <- paste0(makeUrls(start=last.date, end=end, gids=""), "/epg.xml")
-  obs <- XML2Obs(scoreboards) #Note to future self -- using `xpath='//game[@gameday_link]'` for future games gives the RCurl error -- Recv failure: Connection reset by peer
-  obs2 <- obs[grep("^epg//game$", names(obs))]
-  gids <- collapse_obs(obs2)[,"gameday"]
+  scoreboards <- paste0(makeUrls(start=last.date, end=end, gids=""), "/master_scoreboard.json")
+  obs <- lapply(scoreboards, fromJSON)
+  gids <- lapply(obs, function(day) {
+    payload <- day$data$games$game$gameday
```
Comment (cpsievert) [inline R/scrape.R]:
```suggestion
  day$data$games$game$gameday
```

**Comment by cpsievert** [comment]
> I'm still working on making scrape work in 2020.

**Pull request merged**